2025-07-21 15:54:20 | ERROR | error: insert into "activity_logs" ("action", "metadata", "record_id", "table_name", "user_id") values ($1, $2, $3, $4, $5) - insert or update on table "activity_logs" violates foreign key constraint "activity_logs_user_id_foreign"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-07-21 15:54:31 | ERROR | Error: User not found
    at exports.login (D:\React\valuetechsolutions\Backend\src\controllers\authController.js:87:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-21 16:29:12 | ERROR | Error: User not found
    at exports.login (D:\React\valuetechsolutions\Backend\src\controllers\authController.js:87:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-21 19:30:36 | ERROR | Error: Internal error while finding user.
    at exports.login (D:\React\valuetechsolutions\Backend\src\controllers\authController.js:94:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-21 21:02:00 | ERROR | Error: User not found
    at exports.login (D:\React\valuetechsolutions\Backend\src\controllers\authController.js:95:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-23 23:31:16 | ERROR | Error: Bank not found
    at exports.getById (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:22:22)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-23 23:32:32 | ERROR | Error: Bank name and initial are required
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:33:13)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\activityLogger.js:108:5
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\permission.js:10:14
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
2025-07-23 23:32:47 | ERROR | Error: Bank name and initial are required
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:33:13)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\activityLogger.js:108:5
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\permission.js:10:14
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
2025-07-23 23:34:22 | ERROR | TypeError: Cannot read properties of undefined (reading 'toLowerCase')
    at D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:80:47
    at Array.find (<anonymous>)
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:76:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-23 23:35:59 | ERROR | Error: Bank with same name or initial already exists
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:83:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-23 23:38:58 | ERROR | Error: Bank not found
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:72:26)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-23 23:41:09 | ERROR | error: insert into "cities" ("created_at", "created_by", "name", "state_id") values ($1, $2, $3, $4) returning * - insert or update on table "cities" violates foreign key constraint "cities_state_id_foreign"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-07-23 23:44:59 | ERROR | Error: Bank branch not found
    at exports.getById (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:22:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-23 23:48:11 | ERROR | SyntaxError: Expected double-quoted property name in JSON at position 35 (line 3 column 1)
    at JSON.parse (<anonymous>)
    at parse (D:\React\valuetechsolutions\Backend\node_modules\body-parser\lib\types\json.js:77:19)
    at D:\React\valuetechsolutions\Backend\node_modules\body-parser\lib\read.js:123:18
    at AsyncResource.runInAsyncScope (node:async_hooks:211:14)
    at invokeCallback (D:\React\valuetechsolutions\Backend\node_modules\raw-body\index.js:238:16)
    at done (D:\React\valuetechsolutions\Backend\node_modules\raw-body\index.js:227:7)
    at IncomingMessage.onEnd (D:\React\valuetechsolutions\Backend\node_modules\raw-body\index.js:287:7)
    at IncomingMessage.emit (node:events:518:28)
    at endReadableNT (node:internal/streams/readable:1698:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)
2025-07-23 23:54:14 | ERROR | Error: Provided bank_id does not exist
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:40:22)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-23 23:54:59 | ERROR | Error: Provided bank_id does not exist
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:91:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 01:25:12 | ERROR | Error: Bank with same name or initial already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:43:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 06:12:36 | ERROR | Error: User not found
    at exports.login (D:\React\valuetechsolutions\Backend\src\controllers\authController.js:87:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 06:43:26 | ERROR | Error: Bank with same name or initial already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:43:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 06:46:53 | ERROR | Error: Name, bank_id and city_id are required
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:36:13)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\activityLogger.js:108:5
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\permission.js:10:14
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
2025-07-24 06:47:15 | ERROR | Error: Branch with same name already exists for this bank
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:54:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:02:59 | ERROR | Error: Branch with same name already exists for this bank
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:54:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:08:00 | ERROR | Error: Branch with same name already exists for this bank
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:54:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:08:14 | ERROR | Error: Branch with same name already exists for this bank
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:54:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:09:15 | ERROR | Error: Bank with same name or initial already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:43:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:10:43 | ERROR | Error: Bank with same name or initial already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:43:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:10:47 | ERROR | Error: Bank with same name or initial already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:43:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:10:56 | ERROR | TypeError: Cannot destructure property 'name' of 'req.body' as it is undefined.
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:68:13)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\activityLogger.js:108:5
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\beforeUpdateLogger.js:25:5
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:11:03 | ERROR | TypeError: Cannot destructure property 'name' of 'req.body' as it is undefined.
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:68:13)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\activityLogger.js:108:5
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\beforeUpdateLogger.js:25:5
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:11:49 | ERROR | Error: Bank with same name or initial already exists
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:83:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:11:56 | ERROR | Error: Bank with same name or initial already exists
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:83:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:12:01 | ERROR | Error: Bank with same name or initial already exists
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:83:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:13:53 | ERROR | Error: Branch with same name already exists for this bank
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:54:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:16:08 | ERROR | Error: Branch with same name already exists for this bank
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:54:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:20:11 | ERROR | Error: Branch with same name already exists for this bank
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:60:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:20:30 | ERROR | Error: Branch with same name already exists for this bank
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:127:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:20:33 | ERROR | ReferenceError: branch is not defined
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:144:44)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:20:50 | ERROR | ReferenceError: branch is not defined
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:144:44)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:21:45 | ERROR | ReferenceError: branch is not defined
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:144:44)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:24:18 | ERROR | Error: Bank with same name or initial already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:43:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:24:27 | ERROR | Error: Bank with same name or initial already exists
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:83:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:24:31 | ERROR | Error: Bank with same name or initial already exists
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:83:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:24:57 | ERROR | Error: Branch with same name already exists for this bank
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:60:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:31:57 | ERROR | TypeError: State.findByName is not a function
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\stateController.js:34:34)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\activityLogger.js:108:5
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\permission.js:10:14
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
2025-07-24 07:32:12 | ERROR | TypeError: State.findByName is not a function
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\stateController.js:34:34)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\activityLogger.js:108:5
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\permission.js:10:14
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
2025-07-24 07:33:36 | ERROR | Error: State name already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\stateController.js:35:25)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:34:07 | ERROR | Error: State name already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\stateController.js:35:25)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:35:57 | ERROR | Error: State name already exists
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\stateController.js:73:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:39:41 | ERROR | Error: State name already exists
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\stateController.js:73:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:39:46 | ERROR | Error: State name already exists
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\stateController.js:73:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:44:10 | ERROR | Error: City with the same name already exists in this state
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\cityController.js:38:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:44:39 | ERROR | Error: State name already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\stateController.js:35:25)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:45:33 | ERROR | Error: State name already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\cityController.js:42:25)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:47:29 | ERROR | Error: State name already exists
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\stateController.js:73:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:47:56 | ERROR | Error: City with the same name already exists in this state
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\cityController.js:38:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:49:56 | ERROR | Error: City with the same name already exists in this state
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\cityController.js:38:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:53:28 | ERROR | Error: City with the same name already exists in this state
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\cityController.js:38:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:53:37 | ERROR | Error: State name already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\stateController.js:35:25)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 07:53:49 | ERROR | Error: City with the same name already exists in this state
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\cityController.js:38:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 09:32:39 | ERROR | Error: Bank branch not found
    at exports.getById (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:25:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 09:32:39 | ERROR | Error: Bank branch not found
    at exports.getById (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:25:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 09:34:18 | ERROR | Error: Bank branch not found
    at exports.getById (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:25:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 09:34:18 | ERROR | Error: Bank branch not found
    at exports.getById (D:\React\valuetechsolutions\Backend\src\controllers\bankBranchController.js:25:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 09:50:02 | ERROR | Error: Username already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\userController.js:92:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 09:56:00 | ERROR | Error: Username already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\userController.js:92:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-24 10:13:53 | ERROR | ReferenceError: ConflictError is not defined
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\permissionController.js:75:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-25 10:32:44 | ERROR | Error: City with the same name already exists in this state
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\cityController.js:38:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-25 10:37:24 | ERROR | Error: City with the same name already exists in this state
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\cityController.js:39:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-25 10:37:34 | ERROR | Error: City with the same name already exists in this state
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\cityController.js:39:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-25 10:42:29 | ERROR | Error: State not found
    at exports.getById (D:\React\valuetechsolutions\Backend\src\controllers\stateController.js:22:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-25 10:42:29 | ERROR | Error: State not found
    at exports.getById (D:\React\valuetechsolutions\Backend\src\controllers\stateController.js:22:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-25 10:42:42 | ERROR | Error: City with the same name already exists in this state
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\cityController.js:39:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-25 10:45:28 | ERROR | Error: City with the same name already exists in this state
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\cityController.js:39:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-25 10:46:29 | ERROR | Error: City with the same name already exists in this state
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\cityController.js:39:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-25 12:52:19 | ERROR | Error: email already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\userController.js:113:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-25 12:55:47 | ERROR | Error: email already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\userController.js:113:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-25 12:56:57 | ERROR | Error: email already exists
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\userController.js:162:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-25 13:06:46 | ERROR | Error: email already exists
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\userController.js:162:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-25 13:07:02 | ERROR | Error: email already exists
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\userController.js:162:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-25 14:10:35 | ERROR | Error: Username and password are required
    at exports.login (D:\React\valuetechsolutions\Backend\src\controllers\authController.js:81:13)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
    at handle (D:\React\valuetechsolutions\Backend\node_modules\router\index.js:435:11)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at D:\React\valuetechsolutions\Backend\node_modules\router\index.js:295:15
    at processParams (D:\React\valuetechsolutions\Backend\node_modules\router\index.js:582:12)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\index.js:291:5)
    at Function.handle (D:\React\valuetechsolutions\Backend\node_modules\router\index.js:186:3)
2025-07-25 14:11:04 | ERROR | Error: Username and password are required
    at exports.login (D:\React\valuetechsolutions\Backend\src\controllers\authController.js:81:13)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
    at handle (D:\React\valuetechsolutions\Backend\node_modules\router\index.js:435:11)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at D:\React\valuetechsolutions\Backend\node_modules\router\index.js:295:15
    at processParams (D:\React\valuetechsolutions\Backend\node_modules\router\index.js:582:12)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\index.js:291:5)
    at Function.handle (D:\React\valuetechsolutions\Backend\node_modules\router\index.js:186:3)
2025-07-25 14:12:42 | ERROR | TypeError: User.findByEmailOrMobile is not a function
    at exports.login (D:\React\valuetechsolutions\Backend\src\controllers\authController.js:85:29)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
    at handle (D:\React\valuetechsolutions\Backend\node_modules\router\index.js:435:11)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at D:\React\valuetechsolutions\Backend\node_modules\router\index.js:295:15
    at processParams (D:\React\valuetechsolutions\Backend\node_modules\router\index.js:582:12)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\index.js:291:5)
    at Function.handle (D:\React\valuetechsolutions\Backend\node_modules\router\index.js:186:3)
2025-07-26 16:25:12 | ERROR | error: select "category"."id", "category"."name", "category"."is_active", "category"."created_at", "created_user"."name" as "created_by", "category"."updated_at", "updated_user"."name" as "updated_by" from "category" left join "users" as "created_user" on "category"."created_by" = "created_user"."id" left join "users" as "updated_user" on "category"."updated_by" = "updated_user"."id" where "category"."deleted_at" is null and "category"."is_active" = $1 - relation "category" does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-07-26 16:44:21 | ERROR | Error: SubCategory not found
    at exports.getById (D:\React\valuetechsolutions\Backend\src\controllers\subcategoryController.js:24:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-26 16:44:34 | ERROR | Error: Category with the same name already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\categoryController.js:39:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-26 16:45:24 | ERROR | Error: Name and category_id are required
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\subcategoryController.js:36:13)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\activityLogger.js:108:5
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\permission.js:10:14
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
2025-07-26 16:46:46 | ERROR | Error: SubCategory with the same name already exists in this category
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\subcategoryController.js:40:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-26 16:51:12 | ERROR | Error: ChildCategory not found
    at exports.getById (D:\React\valuetechsolutions\Backend\src\controllers\childCategoryController.js:24:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-26 16:51:39 | ERROR | Error: Name and category_id are required
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\subcategoryController.js:36:13)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\activityLogger.js:108:5
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\permission.js:10:14
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
2025-07-26 16:51:53 | ERROR | Error: Name and sub_category_id are required
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\childCategoryController.js:36:13)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\activityLogger.js:108:5
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\permission.js:10:14
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
2025-07-26 17:43:32 | ERROR | Error: Category with the same name already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\categoryController.js:39:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-26 18:14:01 | ERROR | Error: Category with the same name already exists
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\categoryController.js:73:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-26 18:40:17 | ERROR | Error: SubCategory with the same name already exists in this category
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\subcategoryController.js:72:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-26 18:40:32 | ERROR | Error: SubCategory with the same name already exists in this category
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\subcategoryController.js:40:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-26 19:09:37 | ERROR | Error: Name and sub_category_id are required
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\childCategoryController.js:36:13)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\activityLogger.js:108:5
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\permission.js:10:14
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
2025-07-26 19:11:45 | ERROR | Error: Name and sub_category_id are required
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\childCategoryController.js:36:13)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\activityLogger.js:108:5
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\permission.js:10:14
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
2025-07-26 19:39:18 | ERROR | Error: ChildCategory with the same name already exists in this sub-category
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\childCategoryController.js:80:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-26 19:47:59 | ERROR | Error: SubCategory with the same name already exists in this category
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\subcategoryController.js:77:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-26 19:50:39 | ERROR | Error: ChildCategory with the same name already exists in this sub-category
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\childCategoryController.js:80:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-26 19:51:47 | ERROR | Error: State name already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\stateController.js:35:25)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-26 19:52:43 | ERROR | Error: State name already exists
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\stateController.js:73:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-26 20:02:48 | ERROR | Error: City with the same name already exists in this state
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\cityController.js:39:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-26 20:04:30 | ERROR | Error: City with the same name already exists in this state
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\cityController.js:79:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-26 20:06:08 | ERROR | Error: City with the same name already exists in this state
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\cityController.js:79:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-26 20:12:12 | ERROR | Error: User not found
    at exports.login (D:\React\valuetechsolutions\Backend\src\controllers\authController.js:87:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-26 20:12:16 | ERROR | Error: Invalid password
    at exports.login (D:\React\valuetechsolutions\Backend\src\controllers\authController.js:93:13)
2025-07-26 20:14:06 | ERROR | Error: Invalid password
    at exports.login (D:\React\valuetechsolutions\Backend\src\controllers\authController.js:93:13)
2025-07-26 20:14:12 | ERROR | Error: Invalid password
    at exports.login (D:\React\valuetechsolutions\Backend\src\controllers\authController.js:93:13)
2025-07-26 20:19:35 | ERROR | Error: Bank with same name or initial already exists
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\bankController.js:83:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-26 21:17:23 | ERROR | Error: City with the same name already exists in this state
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\cityController.js:39:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-29 15:04:21 | ERROR | ReferenceError: knex is not defined
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\officerController.js:38:15)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\activityLogger.js:108:5
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\permission.js:10:14
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
2025-07-29 15:08:24 | ERROR | ReferenceError: knex is not defined
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\officerController.js:38:15)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\activityLogger.js:108:5
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\permission.js:10:14
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
2025-07-29 15:12:36 | ERROR | TypeError: knex.transaction is not a function
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\officerController.js:39:26)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\activityLogger.js:108:5
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\permission.js:10:14
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
2025-07-29 15:31:43 | ERROR | SyntaxError: Expected double-quoted property name in JSON at position 31 (line 3 column 1)
    at JSON.parse (<anonymous>)
    at parse (D:\React\valuetechsolutions\Backend\node_modules\body-parser\lib\types\json.js:77:19)
    at D:\React\valuetechsolutions\Backend\node_modules\body-parser\lib\read.js:123:18
    at AsyncResource.runInAsyncScope (node:async_hooks:211:14)
    at invokeCallback (D:\React\valuetechsolutions\Backend\node_modules\raw-body\index.js:238:16)
    at done (D:\React\valuetechsolutions\Backend\node_modules\raw-body\index.js:227:7)
    at IncomingMessage.onEnd (D:\React\valuetechsolutions\Backend\node_modules\raw-body\index.js:287:7)
    at IncomingMessage.emit (node:events:518:28)
    at endReadableNT (node:internal/streams/readable:1698:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)
2025-07-29 15:45:13 | ERROR | Error: Name, email, contact number, role_id, department, and branch_id are required.
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\officerController.js:178:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-29 15:46:30 | ERROR | error: select "categories"."id", "categories"."name" from "officer_categories" left join "categories" on "officer_categories"."category_id" = "categories"."id" where "officer_categories"."officer_id" = $1 and "officer_categories"."deleted_at" is null - relation "categories" does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-07-29 15:46:56 | ERROR | error: select "categories"."id", "categories"."name" from "officer_categories" left join "categories" on "officer_categories"."category_id" = "categories"."id" where "officer_categories"."officer_id" = $1 and "officer_categories"."deleted_at" is null - relation "categories" does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-07-29 15:47:03 | ERROR | error: select "categories"."id", "categories"."name" from "officer_categories" left join "categories" on "officer_categories"."category_id" = "categories"."id" where "officer_categories"."officer_id" = $1 and "officer_categories"."deleted_at" is null - relation "categories" does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-07-29 15:49:41 | ERROR | Error: Undefined binding(s) detected when compiling WHERE. Undefined column(s): [email, mobile] query: where "email" = ? or "mobile" = ?
    at QueryCompiler_PG.toSQL (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\query\querycompiler.js:112:13)
    at compileCallback (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\formatter\formatterUtils.js:13:19)
    at rawOrFn (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\formatter\wrappingFormatter.js:225:7)
    at QueryCompiler_PG.whereWrapped (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\query\querycompiler.js:1095:17)
    at QueryCompiler_PG.where (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\query\querycompiler.js:584:34)
    at D:\React\valuetechsolutions\Backend\node_modules\knex\lib\query\querycompiler.js:135:40
    at Array.forEach (<anonymous>)
    at QueryCompiler_PG.select (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\query\querycompiler.js:134:16)
    at QueryCompiler_PG.toSQL (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\query\querycompiler.js:75:29)
    at QueryBuilder_PostgreSQL.toSQL (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\query\querybuilder.js:84:44)
2025-07-29 16:01:51 | ERROR | TypeError: (intermediate value) is not iterable
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\officerController.js:217:30)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-29 16:04:02 | ERROR | Error: The query is empty
    at Client_PG._query (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\dialects\postgres\index.js:217:25)
    at executeQuery (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\internal\query-executioner.js:37:17)
    at Client_PG.query (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\client.js:154:12)
    at D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\transaction.js:375:24
    at new Promise (<anonymous>)
    at trxClient.query (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\transaction.js:370:12)
    at Runner.query (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\runner.js:141:36)
    at ensureConnectionCallback (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\internal\ensure-connection-callback.js:13:17)
    at Runner.ensureConnection (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\runner.js:318:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-29 16:18:54 | ERROR | error: select "users"."id", "users"."name", "users"."email", "users"."mobile", "users"."role_id", "roles"."name" as "role_name", "users"."city_id", "cities"."name" as "city_name", "users"."otp", "users"."otp_expiry", "users"."created_at", "created_user"."name" as "created_by", "users"."updated_at", "updated_user"."name" as "updated_by", "cities"."state_id" as "state_id", "states"."name" as "state_name" from "users" left join "roles" on "users"."role_id" = "roles"."id" left join "cities" on "users"."city_id" = "cities"."id" left join "states" on "cities"."state_id" = "states"."id" left join "users" as "created_user" on "users"."created_by" = "created_user"."id" left join "users" as "updated_user" on "users"."updated_by" = "updated_user"."id" where "users"."id" = $1 and "users"."deleted_at" is null limit $2 - invalid input syntax for type integer: "Owner"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-07-29 16:19:10 | ERROR | error: select "users"."id", "users"."name", "users"."email", "users"."mobile", "users"."role_id", "roles"."name" as "role_name", "users"."city_id", "cities"."name" as "city_name", "users"."otp", "users"."otp_expiry", "users"."created_at", "created_user"."name" as "created_by", "users"."updated_at", "updated_user"."name" as "updated_by", "cities"."state_id" as "state_id", "states"."name" as "state_name" from "users" left join "roles" on "users"."role_id" = "roles"."id" left join "cities" on "users"."city_id" = "cities"."id" left join "states" on "cities"."state_id" = "states"."id" left join "users" as "created_user" on "users"."created_by" = "created_user"."id" left join "users" as "updated_user" on "users"."updated_by" = "updated_user"."id" where "users"."id" = $1 and "users"."deleted_at" is null limit $2 - invalid input syntax for type integer: "Owner"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-07-29 16:19:34 | ERROR | error: select "users"."id", "users"."name", "users"."email", "users"."mobile", "users"."role_id", "roles"."name" as "role_name", "users"."city_id", "cities"."name" as "city_name", "users"."otp", "users"."otp_expiry", "users"."created_at", "created_user"."name" as "created_by", "users"."updated_at", "updated_user"."name" as "updated_by", "cities"."state_id" as "state_id", "states"."name" as "state_name" from "users" left join "roles" on "users"."role_id" = "roles"."id" left join "cities" on "users"."city_id" = "cities"."id" left join "states" on "cities"."state_id" = "states"."id" left join "users" as "created_user" on "users"."created_by" = "created_user"."id" left join "users" as "updated_user" on "users"."updated_by" = "updated_user"."id" where "users"."id" = $1 and "users"."deleted_at" is null limit $2 - invalid input syntax for type integer: "Owner"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-07-29 16:22:03 | ERROR | error: select "users"."id", "users"."name", "users"."email", "users"."mobile", "users"."role_id", "roles"."name" as "role_name", "users"."city_id", "cities"."name" as "city_name", "users"."otp", "users"."otp_expiry", "users"."created_at", "created_user"."name" as "created_by", "users"."updated_at", "updated_user"."name" as "updated_by", "cities"."state_id" as "state_id", "states"."name" as "state_name" from "users" left join "roles" on "users"."role_id" = "roles"."id" left join "cities" on "users"."city_id" = "cities"."id" left join "states" on "cities"."state_id" = "states"."id" left join "users" as "created_user" on "users"."created_by" = "created_user"."id" left join "users" as "updated_user" on "users"."updated_by" = "updated_user"."id" where "users"."id" = $1 and "users"."deleted_at" is null limit $2 - invalid input syntax for type integer: "Owner"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-07-29 16:24:47 | ERROR | error: select "users"."id", "users"."name", "users"."email", "users"."mobile", "users"."role_id", "roles"."name" as "role_name", "users"."city_id", "cities"."name" as "city_name", "users"."otp", "users"."otp_expiry", "users"."created_at", "created_user"."name" as "created_by", "users"."updated_at", "updated_user"."name" as "updated_by", "cities"."state_id" as "state_id", "states"."name" as "state_name" from "users" left join "roles" on "users"."role_id" = "roles"."id" left join "cities" on "users"."city_id" = "cities"."id" left join "states" on "cities"."state_id" = "states"."id" left join "users" as "created_user" on "users"."created_by" = "created_user"."id" left join "users" as "updated_user" on "users"."updated_by" = "updated_user"."id" where "users"."id" = $1 and "users"."deleted_at" is null limit $2 - invalid input syntax for type integer: "Owner"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-07-29 16:25:53 | ERROR | error: select "users"."id", "users"."name", "users"."email", "users"."mobile", "users"."role_id", "roles"."name" as "role_name", "users"."city_id", "cities"."name" as "city_name", "users"."otp", "users"."otp_expiry", "users"."created_at", "created_user"."name" as "created_by", "users"."updated_at", "updated_user"."name" as "updated_by", "cities"."state_id" as "state_id", "states"."name" as "state_name" from "users" left join "roles" on "users"."role_id" = "roles"."id" left join "cities" on "users"."city_id" = "cities"."id" left join "states" on "cities"."state_id" = "states"."id" left join "users" as "created_user" on "users"."created_by" = "created_user"."id" left join "users" as "updated_user" on "users"."updated_by" = "updated_user"."id" where "users"."id" = $1 and "users"."deleted_at" is null limit $2 - invalid input syntax for type integer: "Owner"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-07-29 16:28:45 | ERROR | Error: Undefined binding(s) detected when compiling FIRST. Undefined column(s): [users.id] query: select "users"."id", "users"."name", "users"."email", "users"."mobile", "users"."role_id", "roles"."name" as "role_name", "users"."city_id", "cities"."name" as "city_name", "users"."otp", "users"."otp_expiry", "users"."created_at", "created_user"."name" as "created_by", "users"."updated_at", "updated_user"."name" as "updated_by", "cities"."state_id" as "state_id", "states"."name" as "state_name" from "users" left join "roles" on "users"."role_id" = "roles"."id" left join "cities" on "users"."city_id" = "cities"."id" left join "states" on "cities"."state_id" = "states"."id" left join "users" as "created_user" on "users"."created_by" = "created_user"."id" left join "users" as "updated_user" on "users"."updated_by" = "updated_user"."id" where "users"."id" = ? and "users"."deleted_at" is null limit ?
    at QueryCompiler_PG.toSQL (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\query\querycompiler.js:112:13)
    at QueryBuilder_PostgreSQL.toSQL (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\query\querybuilder.js:84:44)
    at ensureConnectionCallback (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\internal\ensure-connection-callback.js:4:30)
    at Runner.ensureConnection (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\runner.js:318:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Runner.run (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\runner.js:30:19)
    at async exports.update (D:\React\valuetechsolutions\Backend\src\controllers\officerController.js:254:21)
2025-07-29 16:45:27 | ERROR | Error: The query is empty
    at Client_PG._query (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\dialects\postgres\index.js:217:25)
    at executeQuery (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\internal\query-executioner.js:37:17)
    at Client_PG.query (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\client.js:154:12)
    at D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\transaction.js:375:24
    at new Promise (<anonymous>)
    at trxClient.query (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\transaction.js:370:12)
    at Runner.query (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\runner.js:141:36)
    at ensureConnectionCallback (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\internal\ensure-connection-callback.js:13:17)
    at Runner.ensureConnection (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\runner.js:318:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-29 16:50:15 | ERROR | Error: Undefined binding(s) detected when compiling FIRST. Undefined column(s): [roles.id] query: select "roles"."id", "roles"."name", "roles"."created_at", "created_user"."name" as "created_by", "roles"."updated_at", "updated_user"."name" as "updated_by" from "roles" left join "users" as "created_user" on "roles"."created_by" = "created_user"."id" left join "users" as "updated_user" on "roles"."updated_by" = "updated_user"."id" where "roles"."id" = ? and "roles"."deleted_at" is null limit ?
    at QueryCompiler_PG.toSQL (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\query\querycompiler.js:112:13)
    at QueryBuilder_PostgreSQL.toSQL (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\query\querybuilder.js:84:44)
    at ensureConnectionCallback (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\internal\ensure-connection-callback.js:4:30)
    at Runner.ensureConnection (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\runner.js:318:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Runner.run (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\runner.js:30:19)
    at async exports.update (D:\React\valuetechsolutions\Backend\src\controllers\officerController.js:244:18)
2025-07-29 16:52:24 | ERROR | Error: Undefined binding(s) detected when compiling FIRST. Undefined column(s): [roles.id] query: select "roles"."id", "roles"."name", "roles"."created_at", "created_user"."name" as "created_by", "roles"."updated_at", "updated_user"."name" as "updated_by" from "roles" left join "users" as "created_user" on "roles"."created_by" = "created_user"."id" left join "users" as "updated_user" on "roles"."updated_by" = "updated_user"."id" where "roles"."id" = ? and "roles"."deleted_at" is null limit ?
    at QueryCompiler_PG.toSQL (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\query\querycompiler.js:112:13)
    at QueryBuilder_PostgreSQL.toSQL (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\query\querybuilder.js:84:44)
    at ensureConnectionCallback (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\internal\ensure-connection-callback.js:4:30)
    at Runner.ensureConnection (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\runner.js:318:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Runner.run (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\runner.js:30:19)
    at async exports.update (D:\React\valuetechsolutions\Backend\src\controllers\officerController.js:245:18)
2025-07-29 17:58:19 | ERROR | Error: Email or mobile already exists.
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\officerController.js:68:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-29 18:30:39 | ERROR | TypeError: Cannot read properties of undefined (reading 'name')
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\officerController.js:256:26)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-29 18:31:43 | ERROR | TypeError: Cannot read properties of undefined (reading 'name')
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\officerController.js:257:26)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-29 18:35:27 | ERROR | Error: Email already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\officerController.js:149:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-29 18:41:00 | ERROR | Error: Email already exists
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\officerController.js:149:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-31 11:41:06 | ERROR | Error: Mobile number already exists.
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\fieldVerifier\portalOperationsController.js:96:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-31 11:42:25 | ERROR | Error: Email already exists.
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\fieldVerifier\portalOperationsController.js:90:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-31 13:30:25 | ERROR | Error: Email already exists.
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\fieldVerifier\portalOperationsController.js:146:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-07-31 13:31:52 | ERROR | Error: Email already exists.
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\fieldVerifier\portalOperationsController.js:146:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-08-03 09:16:04 | ERROR | Error: Invalid password
    at exports.login (D:\React\valuetechsolutions\Backend\src\controllers\authController.js:93:13)
2025-08-03 09:16:09 | ERROR | error: select "field_verifiers"."id", "field_verifiers"."name", "field_verifiers"."email", "field_verifiers"."mobile", "field_verifiers"."valid_minutes", "field_verifiers"."is_active", "field_verifiers"."created_at", "created_user"."name" as "created_by", "field_verifiers"."updated_at", "updated_user"."name" as "updated_by" from "field_verifiers" left join "users" as "created_user" on "field_verifiers"."created_by" = "created_user"."id" left join "users" as "updated_user" on "field_verifiers"."updated_by" = "updated_user"."id" where "field_verifiers"."deleted_at" is null - relation "field_verifiers" does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-03 09:16:09 | ERROR | error: select "field_verifiers"."id", "field_verifiers"."name", "field_verifiers"."email", "field_verifiers"."mobile", "field_verifiers"."valid_minutes", "field_verifiers"."is_active", "field_verifiers"."created_at", "created_user"."name" as "created_by", "field_verifiers"."updated_at", "updated_user"."name" as "updated_by" from "field_verifiers" left join "users" as "created_user" on "field_verifiers"."created_by" = "created_user"."id" left join "users" as "updated_user" on "field_verifiers"."updated_by" = "updated_user"."id" where "field_verifiers"."deleted_at" is null - relation "field_verifiers" does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-03 09:21:06 | ERROR | error: select "field_verifiers"."id", "field_verifiers"."name", "field_verifiers"."email", "field_verifiers"."mobile", "field_verifiers"."valid_minutes", "field_verifiers"."is_active", "field_verifiers"."created_at", "created_user"."name" as "created_by", "field_verifiers"."updated_at", "updated_user"."name" as "updated_by" from "field_verifiers" left join "users" as "created_user" on "field_verifiers"."created_by" = "created_user"."id" left join "users" as "updated_user" on "field_verifiers"."updated_by" = "updated_user"."id" where "field_verifiers"."deleted_at" is null - column field_verifiers.valid_minutes does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-03 09:21:06 | ERROR | error: select "field_verifiers"."id", "field_verifiers"."name", "field_verifiers"."email", "field_verifiers"."mobile", "field_verifiers"."valid_minutes", "field_verifiers"."is_active", "field_verifiers"."created_at", "created_user"."name" as "created_by", "field_verifiers"."updated_at", "updated_user"."name" as "updated_by" from "field_verifiers" left join "users" as "created_user" on "field_verifiers"."created_by" = "created_user"."id" left join "users" as "updated_user" on "field_verifiers"."updated_by" = "updated_user"."id" where "field_verifiers"."deleted_at" is null - column field_verifiers.valid_minutes does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-03 10:03:35 | ERROR | error: insert into "field_verifiers" ("created_at", "created_by", "email", "mobile", "name", "password", "valid_minutes") values ($1, $2, $3, $4, $5, $6, $7) returning * - column "valid_minutes" of relation "field_verifiers" does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-03 10:04:36 | ERROR | Error: Name, mobile, email, password, and valid_minutes are required.
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\fieldVerifier\portalOperationsController.js:82:13)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\activityLogger.js:108:5
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\permission.js:10:14
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
2025-08-03 10:12:16 | ERROR | error: insert into "field_verifiers" ("city_id", "created_at", "created_by", "email", "mobile", "name", "password", "upi_id") values ($1, $2, $3, $4, $5, $6, $7, $8) returning * - invalid input syntax for type integer: "abc@icic"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-03 12:20:55 | ERROR | Error: Invalid password
    at exports.login (D:\React\valuetechsolutions\Backend\src\controllers\authController.js:93:13)
2025-08-06 18:40:00 | ERROR | Error: Invalid password
    at exports.login (D:\React\valuetechsolutions\Backend\src\controllers\authController.js:93:13)
2025-08-06 18:41:34 | ERROR | Error: username already exists.
    at exports.update (D:\React\valuetechsolutions\Backend\src\controllers\fieldVerifier\portalOperationsController.js:150:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-08-13 12:15:40 | ERROR | Error: Undefined binding(s) detected when compiling FIRST. Undefined column(s): [officers.id] query: select "officers"."id", "officers"."user_id", "officers"."branch_id", "users"."name" as "name", "users"."email" as "email", "users"."mobile" as "mobile", "officers"."is_active", "officers"."created_at", "officers"."updated_at", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "bank_branch"."id" as "branch_id", "bank_branch"."name" as "branch_name" from "officers" left join "users" on "officers"."user_id" = "users"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" where "officers"."deleted_at" is null and "officers"."id" = ? limit ?
    at QueryCompiler_PG.toSQL (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\query\querycompiler.js:112:13)
    at QueryBuilder_PostgreSQL.toSQL (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\query\querybuilder.js:84:44)
    at ensureConnectionCallback (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\internal\ensure-connection-callback.js:4:30)
    at Runner.ensureConnection (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\runner.js:318:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Runner.run (D:\React\valuetechsolutions\Backend\node_modules\knex\lib\execution\runner.js:30:19)
    at async Object.findById (D:\React\valuetechsolutions\Backend\src\models\officer.js:88:21)
    at async exports.create (D:\React\valuetechsolutions\Backend\src\controllers\orderController.js:57:21)
2025-08-13 12:50:36 | ERROR | TypeError: (intermediate value) is not iterable
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\orderController.js:84:21)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-08-13 12:52:48 | ERROR | TypeError: (intermediate value) is not iterable
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\orderController.js:86:21)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-08-14 09:09:41 | ERROR | error: select "officers"."id", "officers"."user_id", "officers"."branch_id", "users"."name" as "name", "users"."email" as "email", "users"."mobile" as "mobile", "officers"."is_active", "officers"."created_at", "officers"."updated_at", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "bank_branch"."id" as "branch_id", "bank_branch"."name" as "branch_name" from "officers" left join "users" on "officers"."user_id" = "users"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" where "officers"."deleted_at" is null and "officers"."id" = $1 limit $2 - invalid input syntax for type integer: "undefined"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-14 09:09:41 | ERROR | error: select "officers"."id", "officers"."user_id", "officers"."branch_id", "users"."name" as "name", "users"."email" as "email", "users"."mobile" as "mobile", "officers"."is_active", "officers"."created_at", "officers"."updated_at", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "bank_branch"."id" as "branch_id", "bank_branch"."name" as "branch_name" from "officers" left join "users" on "officers"."user_id" = "users"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" where "officers"."deleted_at" is null and "officers"."id" = $1 limit $2 - invalid input syntax for type integer: "undefined"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-14 09:11:14 | ERROR | Error: Customer name and contact are required.
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\orderController.js:53:13)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\activityLogger.js:108:5
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\permission.js:10:14
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
2025-08-14 09:13:20 | ERROR | error: select "officers"."id", "officers"."user_id", "officers"."branch_id", "users"."name" as "name", "users"."email" as "email", "users"."mobile" as "mobile", "officers"."is_active", "officers"."created_at", "officers"."updated_at", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "bank_branch"."id" as "branch_id", "bank_branch"."name" as "branch_name" from "officers" left join "users" on "officers"."user_id" = "users"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" where "officers"."deleted_at" is null and "officers"."id" = $1 limit $2 - invalid input syntax for type integer: "undefined"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-14 09:13:20 | ERROR | error: select "officers"."id", "officers"."user_id", "officers"."branch_id", "users"."name" as "name", "users"."email" as "email", "users"."mobile" as "mobile", "officers"."is_active", "officers"."created_at", "officers"."updated_at", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "bank_branch"."id" as "branch_id", "bank_branch"."name" as "branch_name" from "officers" left join "users" on "officers"."user_id" = "users"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" where "officers"."deleted_at" is null and "officers"."id" = $1 limit $2 - invalid input syntax for type integer: "undefined"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-14 09:13:39 | ERROR | error: select "officers"."id", "officers"."user_id", "officers"."branch_id", "users"."name" as "name", "users"."email" as "email", "users"."mobile" as "mobile", "officers"."is_active", "officers"."created_at", "officers"."updated_at", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "bank_branch"."id" as "branch_id", "bank_branch"."name" as "branch_name" from "officers" left join "users" on "officers"."user_id" = "users"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" where "officers"."deleted_at" is null and "officers"."id" = $1 limit $2 - invalid input syntax for type integer: "undefined"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-14 09:13:39 | ERROR | error: select "officers"."id", "officers"."user_id", "officers"."branch_id", "users"."name" as "name", "users"."email" as "email", "users"."mobile" as "mobile", "officers"."is_active", "officers"."created_at", "officers"."updated_at", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "bank_branch"."id" as "branch_id", "bank_branch"."name" as "branch_name" from "officers" left join "users" on "officers"."user_id" = "users"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" where "officers"."deleted_at" is null and "officers"."id" = $1 limit $2 - invalid input syntax for type integer: "undefined"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-14 09:16:19 | ERROR | error: select "officers"."id", "officers"."user_id", "officers"."branch_id", "users"."name" as "name", "users"."email" as "email", "users"."mobile" as "mobile", "officers"."is_active", "officers"."created_at", "officers"."updated_at", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "bank_branch"."id" as "branch_id", "bank_branch"."name" as "branch_name" from "officers" left join "users" on "officers"."user_id" = "users"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" where "officers"."deleted_at" is null and "officers"."id" = $1 limit $2 - invalid input syntax for type integer: "undefined"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-14 09:16:19 | ERROR | error: select "officers"."id", "officers"."user_id", "officers"."branch_id", "users"."name" as "name", "users"."email" as "email", "users"."mobile" as "mobile", "officers"."is_active", "officers"."created_at", "officers"."updated_at", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "bank_branch"."id" as "branch_id", "bank_branch"."name" as "branch_name" from "officers" left join "users" on "officers"."user_id" = "users"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" where "officers"."deleted_at" is null and "officers"."id" = $1 limit $2 - invalid input syntax for type integer: "undefined"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-14 09:16:40 | ERROR | error: select "officers"."id", "officers"."user_id", "officers"."branch_id", "users"."name" as "name", "users"."email" as "email", "users"."mobile" as "mobile", "officers"."is_active", "officers"."created_at", "officers"."updated_at", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "bank_branch"."id" as "branch_id", "bank_branch"."name" as "branch_name" from "officers" left join "users" on "officers"."user_id" = "users"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" where "officers"."deleted_at" is null and "officers"."id" = $1 limit $2 - invalid input syntax for type integer: "undefined"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-14 09:16:41 | ERROR | error: select "officers"."id", "officers"."user_id", "officers"."branch_id", "users"."name" as "name", "users"."email" as "email", "users"."mobile" as "mobile", "officers"."is_active", "officers"."created_at", "officers"."updated_at", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "bank_branch"."id" as "branch_id", "bank_branch"."name" as "branch_name" from "officers" left join "users" on "officers"."user_id" = "users"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" where "officers"."deleted_at" is null and "officers"."id" = $1 limit $2 - invalid input syntax for type integer: "undefined"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-14 09:23:12 | ERROR | Error: Customer name and contact are required.
    at exports.create (D:\React\valuetechsolutions\Backend\src\controllers\orderController.js:53:13)
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\activityLogger.js:108:5
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at D:\React\valuetechsolutions\Backend\src\middleware\permission.js:10:14
    at Layer.handleRequest (D:\React\valuetechsolutions\Backend\node_modules\router\lib\layer.js:152:17)
    at next (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (D:\React\valuetechsolutions\Backend\node_modules\router\lib\route.js:117:3)
2025-08-16 07:16:17 | ERROR | error: insert into "order_comments" ("comment", "commented_at", "order_id", "user_id") values ($1, $2, $3, $4) returning * - relation "order_comments" does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-16 09:17:55 | ERROR | error: select "order_comments"."id", "order_comments"."order_id", "order_comments"."user_id", "users"."name" as "user_name", "order_comments"."comment", "order_comments"."commented_at" from "order_comments" left join "users" on "order_comments"."user_id" = "users"."id" where "order_comments"."order_id" = $1 order by "order_comments"."id" asc - relation "order_comments" does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-16 09:17:55 | ERROR | error: select "order_comments"."id", "order_comments"."order_id", "order_comments"."user_id", "users"."name" as "user_name", "order_comments"."comment", "order_comments"."commented_at" from "order_comments" left join "users" on "order_comments"."user_id" = "users"."id" where "order_comments"."order_id" = $1 order by "order_comments"."id" asc - relation "order_comments" does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-16 09:19:36 | ERROR | error: select "orders"."id", "orders"."order_number", "orders"."customer_name", "orders"."contact", "orders"."alternative_contact", "orders"."officer_id", "officer_user"."name" as "officer_name", "officer_user"."email" as "officer_email", "officer_user"."mobile" as "officer_mobile", "officers"."branch_id", "bank_branch"."name" as "branch_name", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "orders"."manager_id", "manager"."name" as "manager_name", "orders"."registration_number", "orders"."place_of_inspection", "orders"."date_of_inspection", "orders"."current_status_id", "order_status_master"."name" as "current_status_name", "order_status_master"."description" as "current_status_description", "orders"."created_at", "created_user"."name" as "created_by", "orders"."updated_at", "updated_user"."name" as "updated_by" from "orders" left join "order_status_master" on "orders"."current_status_id" = "order_status_master"."id" left join "officers" on "orders"."officer_id" = "officers"."id" left join "users" as "officer_user" on "officers"."user_id" = "officer_user"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" left join "users" as "manager" on "orders"."manager_id" = "manager"."id" left join "users" as "created_user" on "orders"."created_by" = "created_user"."id" left join "users" as "updated_user" on "orders"."updated_by" = "updated_user"."id" where "orders"."deleted_at" is null - relation "orders" does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-16 09:19:36 | ERROR | error: select "orders"."id", "orders"."order_number", "orders"."customer_name", "orders"."contact", "orders"."alternative_contact", "orders"."officer_id", "officer_user"."name" as "officer_name", "officer_user"."email" as "officer_email", "officer_user"."mobile" as "officer_mobile", "officers"."branch_id", "bank_branch"."name" as "branch_name", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "orders"."manager_id", "manager"."name" as "manager_name", "orders"."registration_number", "orders"."place_of_inspection", "orders"."date_of_inspection", "orders"."current_status_id", "order_status_master"."name" as "current_status_name", "order_status_master"."description" as "current_status_description", "orders"."created_at", "created_user"."name" as "created_by", "orders"."updated_at", "updated_user"."name" as "updated_by" from "orders" left join "order_status_master" on "orders"."current_status_id" = "order_status_master"."id" left join "officers" on "orders"."officer_id" = "officers"."id" left join "users" as "officer_user" on "officers"."user_id" = "officer_user"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" left join "users" as "manager" on "orders"."manager_id" = "manager"."id" left join "users" as "created_user" on "orders"."created_by" = "created_user"."id" left join "users" as "updated_user" on "orders"."updated_by" = "updated_user"."id" where "orders"."deleted_at" is null - relation "orders" does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-16 09:19:38 | ERROR | error: select "orders"."id", "orders"."order_number", "orders"."customer_name", "orders"."contact", "orders"."alternative_contact", "orders"."officer_id", "officer_user"."name" as "officer_name", "officer_user"."email" as "officer_email", "officer_user"."mobile" as "officer_mobile", "officers"."branch_id", "bank_branch"."name" as "branch_name", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "orders"."manager_id", "manager"."name" as "manager_name", "orders"."registration_number", "orders"."place_of_inspection", "orders"."date_of_inspection", "orders"."current_status_id", "order_status_master"."name" as "current_status_name", "order_status_master"."description" as "current_status_description", "orders"."created_at", "created_user"."name" as "created_by", "orders"."updated_at", "updated_user"."name" as "updated_by" from "orders" left join "order_status_master" on "orders"."current_status_id" = "order_status_master"."id" left join "officers" on "orders"."officer_id" = "officers"."id" left join "users" as "officer_user" on "officers"."user_id" = "officer_user"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" left join "users" as "manager" on "orders"."manager_id" = "manager"."id" left join "users" as "created_user" on "orders"."created_by" = "created_user"."id" left join "users" as "updated_user" on "orders"."updated_by" = "updated_user"."id" where "orders"."deleted_at" is null - relation "orders" does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-16 09:19:38 | ERROR | error: select "orders"."id", "orders"."order_number", "orders"."customer_name", "orders"."contact", "orders"."alternative_contact", "orders"."officer_id", "officer_user"."name" as "officer_name", "officer_user"."email" as "officer_email", "officer_user"."mobile" as "officer_mobile", "officers"."branch_id", "bank_branch"."name" as "branch_name", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "orders"."manager_id", "manager"."name" as "manager_name", "orders"."registration_number", "orders"."place_of_inspection", "orders"."date_of_inspection", "orders"."current_status_id", "order_status_master"."name" as "current_status_name", "order_status_master"."description" as "current_status_description", "orders"."created_at", "created_user"."name" as "created_by", "orders"."updated_at", "updated_user"."name" as "updated_by" from "orders" left join "order_status_master" on "orders"."current_status_id" = "order_status_master"."id" left join "officers" on "orders"."officer_id" = "officers"."id" left join "users" as "officer_user" on "officers"."user_id" = "officer_user"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" left join "users" as "manager" on "orders"."manager_id" = "manager"."id" left join "users" as "created_user" on "orders"."created_by" = "created_user"."id" left join "users" as "updated_user" on "orders"."updated_by" = "updated_user"."id" where "orders"."deleted_at" is null - relation "orders" does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-16 09:19:39 | ERROR | error: select "orders"."id", "orders"."order_number", "orders"."customer_name", "orders"."contact", "orders"."alternative_contact", "orders"."officer_id", "officer_user"."name" as "officer_name", "officer_user"."email" as "officer_email", "officer_user"."mobile" as "officer_mobile", "officers"."branch_id", "bank_branch"."name" as "branch_name", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "orders"."manager_id", "manager"."name" as "manager_name", "orders"."registration_number", "orders"."place_of_inspection", "orders"."date_of_inspection", "orders"."current_status_id", "order_status_master"."name" as "current_status_name", "order_status_master"."description" as "current_status_description", "orders"."created_at", "created_user"."name" as "created_by", "orders"."updated_at", "updated_user"."name" as "updated_by" from "orders" left join "order_status_master" on "orders"."current_status_id" = "order_status_master"."id" left join "officers" on "orders"."officer_id" = "officers"."id" left join "users" as "officer_user" on "officers"."user_id" = "officer_user"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" left join "users" as "manager" on "orders"."manager_id" = "manager"."id" left join "users" as "created_user" on "orders"."created_by" = "created_user"."id" left join "users" as "updated_user" on "orders"."updated_by" = "updated_user"."id" where "orders"."deleted_at" is null - relation "orders" does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-16 09:19:39 | ERROR | error: select "orders"."id", "orders"."order_number", "orders"."customer_name", "orders"."contact", "orders"."alternative_contact", "orders"."officer_id", "officer_user"."name" as "officer_name", "officer_user"."email" as "officer_email", "officer_user"."mobile" as "officer_mobile", "officers"."branch_id", "bank_branch"."name" as "branch_name", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "orders"."manager_id", "manager"."name" as "manager_name", "orders"."registration_number", "orders"."place_of_inspection", "orders"."date_of_inspection", "orders"."current_status_id", "order_status_master"."name" as "current_status_name", "order_status_master"."description" as "current_status_description", "orders"."created_at", "created_user"."name" as "created_by", "orders"."updated_at", "updated_user"."name" as "updated_by" from "orders" left join "order_status_master" on "orders"."current_status_id" = "order_status_master"."id" left join "officers" on "orders"."officer_id" = "officers"."id" left join "users" as "officer_user" on "officers"."user_id" = "officer_user"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" left join "users" as "manager" on "orders"."manager_id" = "manager"."id" left join "users" as "created_user" on "orders"."created_by" = "created_user"."id" left join "users" as "updated_user" on "orders"."updated_by" = "updated_user"."id" where "orders"."deleted_at" is null - relation "orders" does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-16 09:19:44 | ERROR | error: select "orders"."id", "orders"."order_number", "orders"."customer_name", "orders"."contact", "orders"."alternative_contact", "orders"."officer_id", "officer_user"."name" as "officer_name", "officer_user"."email" as "officer_email", "officer_user"."mobile" as "officer_mobile", "officers"."branch_id", "bank_branch"."name" as "branch_name", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "orders"."manager_id", "manager"."name" as "manager_name", "orders"."registration_number", "orders"."place_of_inspection", "orders"."date_of_inspection", "orders"."current_status_id", "order_status_master"."name" as "current_status_name", "order_status_master"."description" as "current_status_description", "orders"."created_at", "created_user"."name" as "created_by", "orders"."updated_at", "updated_user"."name" as "updated_by" from "orders" left join "order_status_master" on "orders"."current_status_id" = "order_status_master"."id" left join "officers" on "orders"."officer_id" = "officers"."id" left join "users" as "officer_user" on "officers"."user_id" = "officer_user"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" left join "users" as "manager" on "orders"."manager_id" = "manager"."id" left join "users" as "created_user" on "orders"."created_by" = "created_user"."id" left join "users" as "updated_user" on "orders"."updated_by" = "updated_user"."id" where "orders"."deleted_at" is null - relation "orders" does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-16 09:19:44 | ERROR | error: select "orders"."id", "orders"."order_number", "orders"."customer_name", "orders"."contact", "orders"."alternative_contact", "orders"."officer_id", "officer_user"."name" as "officer_name", "officer_user"."email" as "officer_email", "officer_user"."mobile" as "officer_mobile", "officers"."branch_id", "bank_branch"."name" as "branch_name", "bank"."id" as "bank_id", "bank"."name" as "bank_name", "orders"."manager_id", "manager"."name" as "manager_name", "orders"."registration_number", "orders"."place_of_inspection", "orders"."date_of_inspection", "orders"."current_status_id", "order_status_master"."name" as "current_status_name", "order_status_master"."description" as "current_status_description", "orders"."created_at", "created_user"."name" as "created_by", "orders"."updated_at", "updated_user"."name" as "updated_by" from "orders" left join "order_status_master" on "orders"."current_status_id" = "order_status_master"."id" left join "officers" on "orders"."officer_id" = "officers"."id" left join "users" as "officer_user" on "officers"."user_id" = "officer_user"."id" left join "bank_branch" on "officers"."branch_id" = "bank_branch"."id" left join "bank" on "bank_branch"."bank_id" = "bank"."id" left join "users" as "manager" on "orders"."manager_id" = "manager"."id" left join "users" as "created_user" on "orders"."created_by" = "created_user"."id" left join "users" as "updated_user" on "orders"."updated_by" = "updated_user"."id" where "orders"."deleted_at" is null - relation "orders" does not exist
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-16 09:33:36 | ERROR | Error: ChildCategory not found
    at exports.getById (D:\React\valuetechsolutions\Backend\src\controllers\childCategoryController.js:24:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-08-16 09:35:52 | ERROR | error: insert into "orders" ("alternative_contact", "child_category_id", "contact", "created_at", "created_by", "current_status_id", "customer_name", "date_of_inspection", "manager_id", "officer_id", "place_of_inspection", "registration_number") values ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12) returning * - insert or update on table "orders" violates foreign key constraint "orders_current_status_id_foreign"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-16 09:36:18 | ERROR | error: insert into "orders" ("alternative_contact", "child_category_id", "contact", "created_at", "created_by", "current_status_id", "customer_name", "date_of_inspection", "manager_id", "officer_id", "place_of_inspection", "registration_number") values ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12) returning * - insert or update on table "orders" violates foreign key constraint "orders_current_status_id_foreign"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-16 09:36:32 | ERROR | error: insert into "orders" ("alternative_contact", "child_category_id", "contact", "created_at", "created_by", "current_status_id", "customer_name", "date_of_inspection", "manager_id", "officer_id", "place_of_inspection", "registration_number") values ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12) returning * - insert or update on table "orders" violates foreign key constraint "orders_current_status_id_foreign"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-16 09:38:06 | ERROR | error: insert into "orders" ("alternative_contact", "child_category_id", "contact", "created_at", "created_by", "current_status_id", "customer_name", "date_of_inspection", "manager_id", "officer_id", "place_of_inspection", "registration_number") values ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12) returning * - insert or update on table "orders" violates foreign key constraint "orders_current_status_id_foreign"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-16 09:41:44 | ERROR | Error: Order not found
    at exports.getById (D:\React\valuetechsolutions\Backend\src\controllers\orderController.js:29:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-08-16 09:44:24 | ERROR | error: insert into "orders" ("alternative_contact", "child_category_id", "contact", "created_at", "created_by", "current_status_id", "customer_name", "date_of_inspection", "manager_id", "officer_id", "place_of_inspection", "registration_number") values ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12) returning * - insert or update on table "orders" violates foreign key constraint "orders_officer_id_foreign"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-16 09:45:01 | ERROR | error: insert into "orders" ("alternative_contact", "child_category_id", "contact", "created_at", "created_by", "current_status_id", "customer_name", "date_of_inspection", "manager_id", "officer_id", "place_of_inspection", "registration_number") values ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12) returning * - insert or update on table "orders" violates foreign key constraint "orders_officer_id_foreign"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-18 09:25:20 | ERROR | ReferenceError: db is not defined
    at exports.getByOrder (D:\React\valuetechsolutions\Backend\src\controllers\orderCommentController.js:18:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-08-18 09:25:20 | ERROR | ReferenceError: db is not defined
    at exports.getByOrder (D:\React\valuetechsolutions\Backend\src\controllers\orderCommentController.js:18:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-08-18 09:25:36 | ERROR | ReferenceError: db is not defined
    at exports.getByOrder (D:\React\valuetechsolutions\Backend\src\controllers\orderCommentController.js:18:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-08-18 09:25:36 | ERROR | ReferenceError: db is not defined
    at exports.getByOrder (D:\React\valuetechsolutions\Backend\src\controllers\orderCommentController.js:18:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-08-18 09:27:03 | ERROR | ReferenceError: db is not defined
    at exports.getByOrder (D:\React\valuetechsolutions\Backend\src\controllers\orderCommentController.js:18:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-08-18 09:27:08 | ERROR | ReferenceError: db is not defined
    at exports.getByOrder (D:\React\valuetechsolutions\Backend\src\controllers\orderCommentController.js:18:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-08-18 09:27:08 | ERROR | ReferenceError: db is not defined
    at exports.getByOrder (D:\React\valuetechsolutions\Backend\src\controllers\orderCommentController.js:18:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-08-18 09:27:35 | ERROR | ReferenceError: db is not defined
    at exports.getByOrder (D:\React\valuetechsolutions\Backend\src\controllers\orderCommentController.js:18:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-08-18 09:27:35 | ERROR | ReferenceError: db is not defined
    at exports.getByOrder (D:\React\valuetechsolutions\Backend\src\controllers\orderCommentController.js:18:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-08-18 09:29:58 | ERROR | ReferenceError: db is not defined
    at exports.getByOrder (D:\React\valuetechsolutions\Backend\src\controllers\orderCommentController.js:18:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-08-18 09:30:56 | ERROR | ReferenceError: db is not defined
    at exports.getByOrder (D:\React\valuetechsolutions\Backend\src\controllers\orderCommentController.js:18:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-08-18 09:31:09 | ERROR | ReferenceError: db is not defined
    at exports.getByOrder (D:\React\valuetechsolutions\Backend\src\controllers\orderCommentController.js:18:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-08-18 09:38:51 | ERROR | error: insert into "order_comment_tags" ("comment_id", "user_id") values ($1, $2) returning * - invalid input syntax for type integer: "d"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-08-18 10:03:53 | ERROR | error: insert into "order_comment_tags" ("comment_id", "user_id") values ($1, $2), ($3, $4) returning * - invalid input syntax for type integer: "n"
    at Parser.parseErrorMessage (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:285:98)
    at Parser.handlePacket (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:122:29)
    at Parser.parse (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\parser.js:35:38)
    at Socket.<anonymous> (D:\React\valuetechsolutions\Backend\node_modules\pg-protocol\dist\index.js:11:42)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
